tests/handlers/test_base_handler.py

```diff
@@ -1218,114 +1464,143 @@ def get_context(self):
 
 
 class TranslateCoordinatesTestCase(TestCase):
-    def setUp(self, *args, **kwargs):
-        super(TranslateCoordinatesTestCase, self).setUp(*args, **kwargs)
+    def setUp(self):
+        super(TranslateCoordinatesTestCase, self).setUp()
         coords = self.get_coords()
-        self.translate_crop_coordinates = BaseHandler.translate_crop_coordinates(
-            original_width=coords['original_width'],
-            original_height=coords['original_height'],
-            width=coords['width'],
-            height=coords['height'],
-            crop_left=coords['crop_left'],
-            crop_top=coords['crop_top'],
-            crop_right=coords['crop_right'],
-            crop_bottom=coords['crop_bottom']
+        crop_coords = BaseHandler.translate_crop_coordinates(
+            original_width=coords["original_width"],
+            original_height=coords["original_height"],
+            width=coords["width"],
+            height=coords["height"],
+            crop_left=coords["crop_left"],
+            crop_top=coords["crop_top"],
+            crop_right=coords["crop_right"],
+            crop_bottom=coords["crop_bottom"],
         )
+        self.translate_crop_coordinates = crop_coords
 
     def get_coords(self):
         return {
-            'original_width': 3000,
-            'original_height': 2000,
-            'width': 1200,
-            'height': 800,
-            'crop_left': 100,
-            'crop_top': 100,
-            'crop_right': 200,
-            'crop_bottom': 200,
-            'expected_crop': (40, 40, 80, 80)
+            "original_width": 3000,
+            "original_height": 2000,
+            "width": 1200,
+            "height": 800,
+            "crop_left": 100,
+            "crop_top": 100,
+            "crop_right": 200,
+            "crop_bottom": 200,
+            "expected_crop": (40, 40, 80, 80),
         }
 
-    def test_should_be_a_list_of_coords(self):
+    @gen_test
+    async def test_should_be_a_list_of_coords(self):
         expect(self.translate_crop_coordinates).to_be_instance_of(tuple)
 
-    def test_should_translate_from_original_to_resized(self):
-        expect(self.translate_crop_coordinates).to_equal(self.get_coords()['expected_crop'])
+    @gen_test
+    async def test_should_translate_from_original_to_resized(self):
+        expect(self.translate_crop_coordinates).to_equal(
+            self.get_coords()["expected_crop"]
+        )
 
 
 class ImageBadRequestDecompressionBomb(TestCase):
     @classmethod
-    def setUpClass(cls, *args, **kw):
+    def setUpClass(cls):
         cls.root_path = tempfile.mkdtemp()
-        cls.loader_path = abspath(join(dirname(__file__), '../fixtures/images/'))
+        cls.loader_path = abspath(
+            join(dirname(__file__), "../fixtures/images/")
+        )
         cls.base_uri = "/image"
 
     @classmethod
-    def tearDownClass(cls, *args, **kw):
+    def tearDownClass(cls):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method name "tearDownClass" doesn't conform to snake_case naming style](https://app.codacy.com/manual/heynemann/thumbor/pullRequest?prid=4967909)